CITATION.cff

```diff
@@ -264,6 +264,10 @@ preferred-citation:
       family-names: Rockhill
       affiliation: 'University of Oregon, Eugene OR, USA'
       orcid: 'https://orcid.org/0000-0003-3868-7453'
+    - given-names: Julius
+      family-names: Welzel
+      affiliation: 'Department of Neurology, Kiel University, Germany'
+      orcid: 'https://orcid.org/0000-0003-4782-5360'
```

Comment:
this should be in the "authors" section above, not in the `preferred_citation.authors` section here :-) line 178 in this current file looks like the right place.

Reply:
Ups, solved in https://github.com/mne-tools/mne-bids/pull/1255/commits/3a37466eaea68f7d179c600ac75aa4a7c4f7b1b8